core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -238,11 +237,7 @@ private object RTS {
      * @param errors  The effectfully produced list of errors, in reverse order.
      */
     final def dispatchErrors(errors: IO[Void, List[Throwable]]): IO[Void, Unit] =
-      errors.flatMap(
-        // Each error produced by a finalizer must be handled using the
-        // context's unhandled exception handler:
-        _.foldRight(IO.unit[Void])((t, io) => io *> unhandled(t))
-      )
+      errors.flatMap(unhandled)
```

Comment:
At this point, the function probably doesn't need to exist since it's far longer than it's definition.